### DISCUSSION TITLE
请问，能不能像特定动作一样加入特定热键？

### DISCUSSION THREAD

**Discussion by chenxin0397** [shuax/MouseInc.Settings#124] [Ideas]
我现在很喜欢把一些快捷键都设为鼠标手势，当我开始习惯用全局热键的时候就发现原来手势里的一部分命领可以放到热键里，也可以用全局热键来简化一些快捷键，因为是全局热键就会同时影响到其他软件，所以我希望热键功能也能像特定动作一样指定程序，不知这样的期待是否可以实现？

**Comment by shuax** [comment]
规划中

**Reply by chenxin0397** [reply]
期待~！！！
